<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clementine</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="PlaylistView" name="playlist">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="allColumnsShowFocus">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="player_controls">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="clear_playlist_button">
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="back_button">
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="pause_play_button">
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="stop_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="popupMode">
          <enum>QToolButton::MenuButtonPopup</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="forward_button">
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="last_fm_controls" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="Line" name="lastfm_line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="love_button">
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="ban_button">
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="AnalyzerContainer" name="analyzer" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>100</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>36</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Amarok::VolumeSlider" name="volume">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="library_dock">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Library</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="library_filter_clear">
         <property name="icon">
          <iconset>
           <normaloff>:/clear.png</normaloff>:/clear.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="LineEdit" name="library_filter">
         <property name="hint" stdset="0">
          <string>Enter search terms here</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="library_options">
         <property name="icon">
          <iconset>
           <normaloff>:/configure.png</normaloff>:/configure.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="LibraryView" name="library_view">
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragOnly</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="radio_dock">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Radio</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="RadioView" name="radio_view">
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragOnly</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="files_dock">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Files</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="FileView" name="file_view" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMusic">
    <property name="title">
     <string>Music</string>
    </property>
    <addaction name="action_open_media"/>
    <addaction name="separator"/>
    <addaction name="action_previous_track"/>
    <addaction name="action_play_pause"/>
    <addaction name="action_stop"/>
    <addaction name="action_next_track"/>
    <addaction name="separator"/>
    <addaction name="action_love"/>
    <addaction name="action_ban"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuPlaylist">
    <property name="title">
     <string>Playlist</string>
    </property>
    <addaction name="action_add_media"/>
    <addaction name="action_add_stream"/>
    <addaction name="separator"/>
    <addaction name="action_shuffle_mode"/>
    <addaction name="action_repeat_mode"/>
    <addaction name="separator"/>
    <addaction name="action_clear_playlist"/>
    <addaction name="action_shuffle"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="action_configure"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_cover_manager"/>
    <addaction name="action_equalizer"/>
    <addaction name="action_transcode"/>
   </widget>
   <addaction name="menuMusic"/>
   <addaction name="menuPlaylist"/>
   <addaction name="menuTools"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="action_previous_track">
   <property name="icon">
    <iconset>
     <normaloff>:/media-skip-backward.png</normaloff>:/media-skip-backward.png</iconset>
   </property>
   <property name="text">
    <string>Previous track</string>
   </property>
  </action>
  <action name="action_play_pause">
   <property name="icon">
    <iconset>
     <normaloff>:/media-playback-start.png</normaloff>:/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="action_stop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/media-playback-stop.png</normaloff>:/media-playback-stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="action_next_track">
   <property name="icon">
    <iconset>
     <normaloff>:/media-skip-forward.png</normaloff>:/media-skip-forward.png</iconset>
   </property>
   <property name="text">
    <string>Next track</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset>
     <normaloff>:/exit.png</normaloff>:/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_stop_after_this_track">
   <property name="icon">
    <iconset>
     <normaloff>:/media-playback-stop.png</normaloff>:/media-playback-stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop after this track</string>
   </property>
  </action>
  <action name="filter_age_all">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Entire collection</string>
   </property>
  </action>
  <action name="filter_age_today">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Added today</string>
   </property>
  </action>
  <action name="filter_age_week">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Added this week</string>
   </property>
  </action>
  <action name="filter_age_three_months">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Added within three months</string>
   </property>
   <property name="toolTip">
    <string>Added within three months</string>
   </property>
  </action>
  <action name="filter_age_year">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Added this year</string>
   </property>
  </action>
  <action name="filter_age_month">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Added this month</string>
   </property>
  </action>
  <action name="action_love">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/last.fm/love.png</normaloff>:/last.fm/love.png</iconset>
   </property>
   <property name="text">
    <string>Love</string>
   </property>
  </action>
  <action name="action_ban">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/last.fm/ban.png</normaloff>:/last.fm/ban.png</iconset>
   </property>
   <property name="text">
    <string>Ban</string>
   </property>
  </action>
  <action name="action_clear_playlist">
   <property name="icon">
    <iconset>
     <normaloff>:/clear-list.png</normaloff>:/clear-list.png</iconset>
   </property>
   <property name="text">
    <string>Clear playlist</string>
   </property>
   <property name="toolTip">
    <string>Clear playlist</string>
   </property>
  </action>
  <action name="action_edit_track">
   <property name="icon">
    <iconset>
     <normaloff>:/edit-track.png</normaloff>:/edit-track.png</iconset>
   </property>
   <property name="text">
    <string>Edit track information...</string>
   </property>
  </action>
  <action name="action_renumber_tracks">
   <property name="text">
    <string>Renumber tracks in this order...</string>
   </property>
  </action>
  <action name="action_selection_set_value">
   <property name="text">
    <string>Set value for all selected tracks...</string>
   </property>
  </action>
  <action name="action_edit_value">
   <property name="text">
    <string>Edit tag...</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="action_configure">
   <property name="icon">
    <iconset>
     <normaloff>:/configure.png</normaloff>:/configure.png</iconset>
   </property>
   <property name="text">
    <string>Configure Clementine...</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About Clementine...</string>
   </property>
  </action>
  <action name="action_shuffle">
   <property name="icon">
    <iconset>
     <normaloff>:/shuffle.png</normaloff>:/shuffle.png</iconset>
   </property>
   <property name="text">
    <string>Shuffle playlist</string>
   </property>
  </action>
  <action name="action_add_media">
   <property name="icon">
    <iconset>
     <normaloff>:/open_media.png</normaloff>:/open_media.png</iconset>
   </property>
   <property name="text">
    <string>Add media...</string>
   </property>
  </action>
  <action name="action_add_stream">
   <property name="icon">
    <iconset>
     <normaloff>:/open_stream.png</normaloff>:/open_stream.png</iconset>
   </property>
   <property name="text">
    <string>Add stream...</string>
   </property>
  </action>
  <action name="action_open_media">
   <property name="icon">
    <iconset>
     <normaloff>:/open_media.png</normaloff>:/open_media.png</iconset>
   </property>
   <property name="text">
    <string>Open media...</string>
   </property>
  </action>
  <action name="action_cover_manager">
   <property name="icon">
    <iconset>
     <normaloff>:/download.png</normaloff>:/download.png</iconset>
   </property>
   <property name="text">
    <string>Cover Manager</string>
   </property>
  </action>
  <action name="action_shuffle_mode">
   <property name="icon">
    <iconset>
     <normaloff>:/media-playlist-shuffle.png</normaloff>:/media-playlist-shuffle.png</iconset>
   </property>
   <property name="text">
    <string>Shuffle mode</string>
   </property>
  </action>
  <action name="action_repeat_mode">
   <property name="icon">
    <iconset>
     <normaloff>:/media-playlist-repeat.png</normaloff>:/media-playlist-repeat.png</iconset>
   </property>
   <property name="text">
    <string>Repeat mode</string>
   </property>
  </action>
  <action name="action_remove_from_playlist">
   <property name="icon">
    <iconset>
     <normaloff>:/list-remove.png</normaloff>:/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove from playlist</string>
   </property>
  </action>
  <action name="group_by_artist">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Group by Artist</string>
   </property>
  </action>
  <action name="group_by_artist_album">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Group by Artist/Album</string>
   </property>
  </action>
  <action name="group_by_artist_yearalbum">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Group by Artist/Year - Album</string>
   </property>
  </action>
  <action name="group_by_album">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Group by Album</string>
   </property>
  </action>
  <action name="group_by_genre_album">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Group by Genre/Album</string>
   </property>
  </action>
  <action name="group_by_genre_artist_album">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Group by Genre/Artist/Album</string>
   </property>
  </action>
  <action name="group_by_advanced">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Advanced grouping...</string>
   </property>
  </action>
  <action name="action_equalizer">
   <property name="text">
    <string>Equalizer</string>
   </property>
  </action>
  <action name="action_transcode">
   <property name="text">
    <string>Transcode Music</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Amarok::VolumeSlider</class>
   <extends>QSlider</extends>
   <header>sliderwidget.h</header>
  </customwidget>
  <customwidget>
   <class>PlaylistView</class>
   <extends>QTreeView</extends>
   <header>playlistview.h</header>
  </customwidget>
  <customwidget>
   <class>LibraryView</class>
   <extends>QTreeView</extends>
   <header>libraryview.h</header>
  </customwidget>
  <customwidget>
   <class>FileView</class>
   <extends>QWidget</extends>
   <header>fileview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RadioView</class>
   <extends>QTreeView</extends>
   <header>radioview.h</header>
  </customwidget>
  <customwidget>
   <class>AnalyzerContainer</class>
   <extends>QWidget</extends>
   <header>analyzers/analyzercontainer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
